public/app/features/alerting/unified/hooks/useAbilities.ts

```diff
@@ -91,6 +91,12 @@ export enum AlertRuleAction {
   DeletePermanently = 'delete-alert-rule-permanently',
 }
 
+// this enum list all of the bulk actions we can perform on a folder
+export enum FolderAction {
```

Comment:
Could we be more precise in naming here? How about `BulkFolderAction`
And then `PauseRules`, `DeleteRules`?
The current name suggests we try to pause or delete a folder, not rules in it